.github/workflows/upload_oci.yml

```diff
@@ -75,10 +75,19 @@ jobs:
       - name: Add additional semver tag
         run: |
           echo ${{ secrets.GITHUB_TOKEN }} | oras login -u ${{ github.repository_owner }} --password-stdin ghcr.io
+          COMMIT_SHA_SHORT=$(git rev-parse HEAD | cut -c1-8)
+          # Keep CNAME reference
           CNAME2=${CNAME//_/-}
+          # Remove version from CNAME
           CNAME2=${CNAME2//-${{ inputs.version }}/}
-          echo "Adding additional tag: ${{ inputs.upload_version || inputs.version }}-${CNAME2}-${{ matrix.arch }}"
-          oras tag ghcr.io/${{ github.repository }}:${{ inputs.version }}-${CNAME}-${{ matrix.arch }} ${{ inputs.upload_version || inputs.version }}-${CNAME2}-${{ matrix.arch }}
+          # Remove commit sha from CNAME
+          CNAME2=${CNAME2//-${COMMIT_SHA_SHORT}/}
+          # 8 charachter commit hash
+          DASHED_VERSION=${{ inputs.upload_version || inputs.version }}
+          DASHED_VERSION=${DASHED_VERSION//./-}
+
+          echo "Adding additional tag: ${{ inputs.upload_version || inputs.version }}-${CNAME2}-${DASHED_VERSION}-${COMMIT_SHA_SHORT}"
```

Comment:
Similar adjustments required [here](https://github.com/gardenlinux/gardenlinux-ccloud/blob/8223575e6602bc5c62a3642e8fc832d518517e67/.github/workflows/test.yml#L36) and [there](https://github.com/gardenlinux/gardenlinux-ccloud/blob/8223575e6602bc5c62a3642e8fc832d518517e67/.github/workflows/dev.yml#L86).

Reply:
ah, these follow the same tag, should we keep them in line with the original tag e.g. [2058.0.0-metal-sci_usi-amd64-2058.0.0-8223575e-amd64](https://github.com/orgs/gardenlinux/packages/container/gardenlinux-ccloud/580275598?tag=2058.0.0-metal-sci_usi-amd64-2058.0.0-8223575e-amd64)?

Reply:
ah no, we want to follow the pr-NUM tagging...